src/components/tree/tree.stories.ts

```diff
@@ -61,6 +61,22 @@ export const simple = (): string => html`
   </calcite-tree>
 `;
 
+export const selectionModeNoneScales = stepStory(
```

Comment:
Per our convo, can you replace this with a single test-only story that doesn't set scale (default)?